doc/tutorials/spark_estimator.rst

```diff
@@ -51,17 +63,141 @@ type or spark array type.
 - "raw_prediction": represents the output margin values.
 - "probability": represents the prediction probability on each label.
+
+
+***************************
+XGBoost PySpark GPU support
+***************************
+
+XGBoost PySpark supports GPU training and prediction. To enable GPU support, you first need
+to install the xgboost and cudf packages. Then you can set `use_gpu` parameter to `True`.
+
+Below tutorial will show you how to train a model with XGBoost PySpark GPU on Spark
+standalone cluster.
+
+
+Write your PySpark application
+==============================
+
+.. code-block:: python
+
+  from xgboost.spark import SparkXGBRegressor
+  spark = SparkSession.builder.getOrCreate()
+
+  # read data into spark dataframe
+  train_data_path = "xxxx/train"
+  train_df = spark.read.parquet(data_path)
+
+  test_data_path = "xxxx/test"
+  test_df = spark.read.parquet(test_data_path)
+
+  # assume the label column is named "class"
+  label_name = "class"
+
+  # get a list with feature column names
+  feature_names = [x.name for x in train_df.schema if x.name != label]
+
+  # create a xgboost pyspark regressor estimator and set use_gpu=True
+  regressor = SparkXGBRegressor(
+    features_col=feature_names,
+    label_col=label_name,
+    num_workers=2,
+    use_gpu=True,
+  )
+
+  # train and return the model
+  model = regressor.fit(train_df)
+
+  # predict on test data
+  predict_df = model.transform(test_df)
+  predict_df.show()
+
+Prepare the necessary packages
+==============================
+
+We recommend using Conda or Virtualenv to manage python dependencies
+in PySpark. Please refer to
+`How to Manage Python Dependencies in PySpark <https://www.databricks.com/blog/2020/12/22/how-to-manage-python-dependencies-in-pyspark.html>`_.
+
+.. code-block:: bash
+
+  conda create -y -n xgboost-env -c conda-forge conda-pack python=3.9
+  conda activate xgboost-env
+  pip install xgboost
+  pip install cudf
+  conda pack -f -o xgboost-env.tar.gz
+
+
+Submit the PySpark application
+==============================
+
+Assuming you have configured your Spark cluster with GPU support, if not yet, please
+refer to `spark standalone configuration with GPU support <https://nvidia.github.io/spark-rapids/docs/get-started/getting-started-on-prem.html#spark-standalone-cluster>`_.
+
+.. code-block:: bash
+
+  export PYSPARK_DRIVER_PYTHON=python
+  export PYSPARK_PYTHON=./environment/bin/python
+
+  spark-submit \
+    --master spark://<master-ip>:7077 \
+    --conf spark.executor.resource.gpu.amount=1 \
+    --conf spark.task.resource.gpu.amount=1 \
+    --archives xgboost-env.tar.gz#environment \
+    xgboost_app.py
+
+
+Model Persistence
+=================
+
+.. code-block:: python
+
+  # save the model
+  model.save("/tmp/xgboost-pyspark-model")
+
+  # load the model
+  model2 = SparkXGBRankerModel.load("/tmp/xgboost-pyspark-model")
+
+The above code snippet shows how to save/load xgboost pyspark model. And you can also
```

Comment:
I strongly prefer using the booster attribute and would like to keep this special file name as a workaround that should be used sparingly.

Reply:
Hmm, this is the standard spark way to save/load model. I can also mention the booster attribute.